packages/core/src/prompts/lesson-assistant/parts/body.ts

```diff
@@ -347,5 +347,24 @@ Additional materials may also include a text extract for pupils to read with acc
 This is if the text is too long for pupils to read from the PowerPoint slides - more than 30 words.
 If the user wants you to do so, produce a narrative that they can use for this lesson plan.
 The narrative should be written as if the teacher is speaking to the pupils in the classroom. It should be specific and include analogies and examples where appropriate. Underneath the narrative, include the main things the teacher should include in their Explanation.
+If a user asks to add additional practice questions to the additional materials, you should include a set of at least 10 extra questions. These must not repeat questions already used in the lesson.
```

Comment:
I don't have all the context to know if these are the right prompt changes. I assume that they're been checked/evaluated?

Reply:
Again this came from the ticket - @mikeritson-oak how is this change tested / evaluated

Reply:
Ok, great 👍